src/main/java/thePackmaster/patches/OpeningRunScreenPatch.java

```diff
@@ -29,7 +29,7 @@ public class OpeningRunScreenPatch {
     )
     public static void SetTheThing(AbstractEvent __instance) {
         if(!Settings.isEndless || AbstractDungeon.floorNum <= 1) {
-            if (AbstractDungeon.player.chosenClass == ThePackmaster.Enums.THE_PACKMASTER) {
+            if (AbstractDungeon.player.chosenClass == ThePackmaster.Enums.THE_PACKMASTER && SpireAnniversary5Mod.currentPoolPacks.isEmpty()) {
                 if (SpireAnniversary5Mod.allPacksMode) {
                     SpireAnniversary5Mod.logger.info("Vex's All Packs Override Enabled. Skipping intro screen");
                     SpireAnniversary5Mod.currentPoolPacks.clear();
```

Comment:
There are a small number of `SpireAnniversary5Mod.currentPoolPacks.clear();` calls like this one that could now be removed (another is in `startOfGamePackSetup` in the main mod file). They're harmless but unnecessary as the code is now written, since the list of packs is guaranteed to be empty by the time you get there.

I didn't remove this call (or the others like it) because I wanted to reduce the chance of issues if this logic gets moved around or changed. Also I like keeping changesets minimal, so that it's clear what the necessary changes are to achieve the goal (vs. optional stuff/cleanup).